specification/iothub/resource-manager/Microsoft.Devices/preview/2022-04-30-preview/iothub.json

```diff
@@ -4239,7 +4358,7 @@
           "type": "string"
         },
         "inputBlobName": {
-          "description": "The blob name to be used when importing from the provided input blob container.",
+          "description": "from the provided input blob container.",
```

Comment:
why the change on description? seems the original one is better?

Reply:
Good catch, it will be reverted.